bundle.md

```diff
@@ -1,6 +1,6 @@
-# Filesystem Bundle
+# <a name="filesystemBundle" />Filesystem Bundle
 
-## Container Format
+## <a name="containerFormat" />Container Format
```

Comment:
I don't think link stability is a major issue, because folks referencing the spec should be pinning their reference to a particular version ([like this][1]).  If we change the header later on, that link (pinned to v1.0.0-rc2) will still work.  Even if you do attempt to provide a stable anchor between spec releases, sometimes the section will go away, or get moved to a different file, or be split into two sections, or whatever.  In all of those cases the old unpinned link will no longer work regardless of any attempt at freezing the anchor.  Do we really expect lots of churn in header text but minimal churn in the section or file structure?

That said, if folks do expect significant churn in header text and are happy to nip that in the bud by adding these headers to all of our existing sections, more power to 'em ;).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md#container-format